test/utils.js

```diff
@@ -1,16 +1,20 @@
-// eslint-disable-next-line camelcase
-async function deploySystem(deploy_list) {
-  const deployed = {} // name: contract object
-  const linkable = {} // name: linkable address
+// FIXME Retrieves past events. This is a workaround for a known issue described
+// FIXME here: https://github.com/nomiclabs/hardhat/pull/1163
+// FIXME The preferred way of getting events would be using listners:
+// FIXME https://docs.ethers.io/v5/api/contract/contract/#Contract--events
```

Comment:
Maybe just:
```
// FIXME: Abcdef 
//        ghijklm
```

Reply:
Fixed in https://github.com/keep-network/sortition-pools/pull/127/commits/4ba45853abf00e3486568229a9bbddac8475f0bb.